client/src/services/api.js

```diff
@@ -20,6 +26,18 @@ class Api {
     this.apiUrl = 'http://localhost:8080/api/';
   }
 
+
+  // eslint-disable-next-line no-unused-vars
+  getLogedin(_email, _password) {
```

Comment:
https://github.com/Social-projects-Rivne/Lunch_Time/pull/36/commits/1bd8545e3a423874d47d0cb2a51d777215ad38c1

Reply:
Agree. We have such method already. Just pass you endpoint.